### ISSUE TITLE
Please add a dependency on json

### ISSUE THREAD

**Issue by halbtuerke** [defunkt/github-gem#10]
It seems that the GitHub gem needs the json gem to work properly, as I had some issues when I installed it yesterday:

```
/System/Library/Frameworks/Ruby.framework/Versions/1.8/usr/lib/ruby/1.8/rubygems      /custom_require.rb:31:in `gem_original_require': no such file to load -- json (LoadError)
from /System/Library/Frameworks/Ruby.framework/Versions/1.8/usr/lib/ruby/1.8/rubygems/custom_require.rb:31:in `require'
from /Library/Ruby/Gems/1.8/gems/github-0.4.0/bin/../lib/github.rb:9
from /System/Library/Frameworks/Ruby.framework/Versions/1.8/usr/lib/ruby/1.8/rubygems/custom_require.rb:31:in `gem_original_require'
from /System/Library/Frameworks/Ruby.framework/Versions/1.8/usr/lib/ruby/1.8/rubygems/custom_require.rb:31:in `require'
from /Library/Ruby/Gems/1.8/gems/github-0.4.0/bin/github:6
from /usr/bin/github:19:in `load'
from /usr/bin/github:19
```

**Comment by mikeymckay** [comment]
This was just reported to me as well from a colleague installing github on OSX.

**Comment by pnutus** [comment]
I second (third?) this.

**Comment by drnic** [comment]
added json dependency. Closed by 5e40c92cc4396d26561f36037fc68343f714eea4 and #11
